Vision/SocketHandlerTask.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.Vision;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonStreamParser;
+
+public class SocketHandlerTask extends Thread {
+
+    public Socket socket;
+    public boolean isFront;
+
+    Gson gson;
+    JsonStreamParser parser;
+
+    VisionData visionData;
+
+    public SocketHandlerTask(Socket socket) {
+        super();
+        this.socket = socket;
+        gson = new Gson();
+<<<<<<< HEAD
```

Comment:
The <<<<< indicates some sort of rebase problems.
This will not compile

Reply:
yeah when pushing this it was very late and i didn't get to make a read me file or merge the code